packages/opencensus-core/src/tags/types.ts

```diff
@@ -25,3 +25,38 @@ export interface TagValue {
   /** The value of a tag. */
   readonly value: string;
 }
+
+/** TagValueWithMetadata holds a TagValue and a TagMetadata. */
+export interface TagValueWithMetadata {
```

Comment:
If possible please don't expose this in the public APIs. In Java it's used as an internal implementation class (not part of API).

Reply:
Done, now this ```interface``` is limited to core package only.